sympy/logic/boolalg.py

```diff
@@ -1637,6 +1808,84 @@ def POSform(variables, minterms, dontcares=None):
     return And(*[_convert_to_varsPOS(x, variables) for x in essential])
 
 
+def ANF(variables, truthvalues):
+    """
+    The ANF function converts the list of truth values to
+    Algebraic Normal Form (ANF).
+
+    The variables must be given as the first argument.
+
+    Return True, False, logical And function (i.e., the
+    "Zhegalkin monomial") or logical Xor function (i.e.,
+    the "Zhegalkin polynomial").
```

Comment:
Maybe point out that this is because when True and False are represented by 1 and 0, respectively, then And is multiplication and Xor is addition.